src/Checkout.vue

```diff
@@ -52,6 +52,11 @@ export default {
       type: String,
       default: undefined
     },
+    shippingIn: {
+      type: Array,
+      default: [],
+      validator: (value) => !['AS', 'CX', 'CC', 'CU', 'HM', 'IR', 'KP', 'MH', 'FM', 'NF', 'MP', 'PW', 'SD', 'SY', 'UM', 'VI'].includes(value)
```

Comment:
Adjust the validator. Put the array of allowedCountries in the `constants.js` see `SUPPORTED_LANGS` implementation.